oe_content.info.yml

```diff
@@ -16,6 +16,7 @@ config_devel:
   install:
     - field.storage.node.oe_author
     - field.storage.node.oe_publication_date
+    - field.storage.node.oe_reference
```

Comment:
This is wrong right? the reference field shouldn't be exported on the main module.